person.rb

```diff
@@ -26,4 +28,12 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Also here, for the same comment I left in the `add_rental` method of the book class, we need to assert the `'has man` y relationship. For that, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
> Something like...
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```